samples/ai-meeting-helper/README.md

```diff
@@ -0,0 +1,127 @@
+---
+page_type: sample
+description: This sample showcases how to utilize Azure Open AI Search within Teams Toolkit for Visual Studio Code to extract Action Items and Meeting summaries from meeting transcriptions.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
+
+# Compliance Checker with Azure AI Search sample
+
+## Interaction with app
+
+ ![ai-meeting-helperGif](Images/ai-meeting-helperGif.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
```

Comment:
Sample.json file is also missing in this sample

Reply:
Done